src/components/nav/SlideNumber.jsx

```diff
@@ -1,37 +1,29 @@
-import React, { useState, useEffect } from 'react';
+import React, { useEffect } from "react";
 import { PREFIX } from "../../constants/classes-names";
 
-const SlideNumber = (
-    {
-        fsLightbox: {
-            componentsServices,
-            props: { sources },
-            stageIndexes
-        }
-    }
-) => {
-    const [slideNumber, setSlideNumber] = useState(stageIndexes.current + 1);
-    componentsServices.setSlideNumber = (number) => {
-        setSlideNumber(number);
-    };
-
-    const slideNumberOuter = React.createRef();
-    const slideNumberInner = React.createRef();
+const SlideNumber = ({
+  fsLightbox: {
+    props: { sources },
+  },
+  stageIndex,
+}) => {
+  const slideNumberOuter = React.createRef();
+  const slideNumberInner = React.createRef();
 
-    useEffect(() => {
-        if (slideNumberInner.current.offsetWidth > 55) {
-            slideNumberOuter.current.style.justifyContent = 'flex-start';
-        }
-    }, []);
+  useEffect(() => {
+    if (slideNumberInner.current.offsetWidth > 55) {
+      slideNumberOuter.current.style.justifyContent = "flex-start";
+    }
+  }, []);
 
-    return (
-        <div ref={slideNumberOuter} className={`${PREFIX}slide-number-container`}>
-            <div ref={slideNumberInner} className="fslightbox-flex-centered">
-                <span data-test-id="slide-number">{slideNumber}</span>
-                <span className={`${PREFIX}slash`} />
-                <span>{sources.length}</span>
-            </div>
-        </div>
-    );
+  return (
+    <div ref={slideNumberOuter} className={`${PREFIX}slide-number-container`}>
+      <div ref={slideNumberInner} className="fslightbox-flex-centered">
+        <span data-test-id="slide-number">{stageIndex + 1}</span>
```

Comment:
Curious why you had to add one to this?

Reply:
StageIndex isthe actual index of the array. The slidenumber on the top left should not show eg 0/5 ect, so had to +1